### ISSUE TITLE
Two problems about the discriminator network

### ISSUE THREAD

**Issue by suntao498253097** [imatge-upc/saliency-salgan-2017#19]
Dear friend,
  I have two problems about the discriminator network : Firstly, I found that the drop of discriminator network's loss was very slow when I was training salgan.  Secondly, I found the model size of discriminator network I trained was different from yours.(5.5MB vs.3.4MB)  why?  Did you have done some change to your discriminator?
  Can you give me some suggestion? I am looking forward to your reply.Thanks.
                                                                                                                                             Sincerely yours

**Comment by junting** [comment]
Hi @suntao498253097 ,

The drop of discriminator loss should be very slow, otherwise it can be very unstable. But you can always change the hyperparamers to accelerate the training.
I have just make a minor change in the code now. Could you please try if you can use the pretrained model? And also I suggest you to load both model and check the number of elements in it, to make sure they belong to the same model architecture. 

Best,
